Remove side menu for tutorials; add section texts (#417)
toitware/web-docs: docs/tutorials/index.mdx

</Boxes>

### Power

> This section provides an introduction to power management.

Reviewer: I think I prefer not referring to the sections. 

```suggestion
Learn how power management works and save precious battery life.
```

</Boxes>

## Network
Get to know about how to use the network capabilities of your device.

<Boxes>
<Box title="HTTP" to="network/http">
<img
  src={Http}
  alt="HTTP logo."
/>

Connect to the internet with HTTP.
</Box>

<Box title="MQTT" to="mqtt">
Send data to an MQTT broker.
</Box>

</Boxes>

## Common tasks

> This section provides tutorials on how to carry out common tasks such as

Reviewer: ```suggestion
Learn how to carry out common tasks such as
```
